app/controllers/idv/review_controller.rb

```diff
@@ -68,5 +76,13 @@ def submit_applicant
       idv_attempter.increment
       resolution
     end
+
+    def password
+      params.require(:user)[:password]
```

Comment:
Should this be either `.require(:password)[:password]` or `.require(user: :password)[:user][:password]` ?